superflore/generate_installers.py

```diff
@@ -0,0 +1,84 @@
+# Copyright 2017 Open Source Robotics Foundation, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from rosinstall_generator.distro import get_distro
+from rosinstall_generator.distro import get_package_names
+
+from superflore.utils import err
+from superflore.utils import get_pkg_version
+from superflore.utils import info
+from superflore.utils import ok
+from superflore.utils import warn
+
+
+def generate_installers(
+    distro_name,      # ros distro name
+    overlay,          # repo instance
+    gen_pkg_func,     # function to call for generating
+    update=True       # are we in update mode?
```

Comment:
What does update mean here? I thought it was forcing an update but https://github.com/ros-infrastructure/superflore/pull/76/files#diff-22c1a4ad79cb71ae0b51b9ef9621bf80R60 doesn't make sense in that case.

Reply:
Ah, `update` gets set to `True` to keep the ebuilds that are currently there (so quite the opposite). I'll change it back to `preserve_existing`, which it was previously.